k8s/environments/prod/common-overlay/kustomization.yaml

```diff
@@ -1,6 +1,8 @@
 ---
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
+resources:
```

Comment:
don't add it to prod yet until we get the connectivity working this won't start up, we can add it later